extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzBuildTimeConfig.java

```diff
@@ -74,6 +74,19 @@ public class QuartzBuildTimeConfig {
     @ConfigItem
     public Optional<String> selectWithLockSql;
 
+    /**
+     * Allows to specify fully qualified class name for a custom JDBC driver delegate.
```

Comment:
I think that you need to add a grammatical [patient](https://en.wikipedia.org/wiki/Patient_%28grammar%29) in this sentence. Something like _"Allows users to specify..."_ ;-).

Reply:
> grammatical [patient](https://en.wikipedia.org/wiki/Patient_%28grammar%29)

The what now? :eyes:
That is an extremely fancy way of saying my grammar processor went haywire :)

Reply:
Fixed